### DISCUSSION TITLE
It is possible to apply Scrubber to Json property names?

### DISCUSSION THREAD

**Discussion by Dreamescaper** [VerifyTests/Verify#1235] [Q&A]
Hi all,

I have a Dictionary in my response type, therefore a serialized JSON has dynamic property names. I wanted to apply Scrubber to replace them with some placeholder, but, apparently, Scrubbers are only applied to Json values, but not to keys.

It is possible to force Scubber on keys as well?

**Comment by SimonCropp** [comment]
not current. i would consider a Pull Request
